### ISSUE TITLE
Big Sur purge takes very long time to delete

### ISSUE THREAD

**Issue by dmuiX** [canonical/multipass#1982]
multipass purge takes very long time to delete
-vvvv is not showing anything. I don't see any output here. So i don't know whats going on and how long it takes to purge all. Maybe it takes a very long time? If yes it should say something about that.

**Comment by Saviq** [comment]
Hi @dmuiX, no it shouldn't take a long time at all.

Which driver are you using? The default hyperkit, or VirtualBox?

What's your `multipass info --all` output?

**Comment by dmuiX** [comment]
hyperkit. 
Yeah thats not working at all. because of the bug I have reported^^. I will print it if i have access to multipassd again :D

**Comment by dmuiX** [comment]
Could it be that it is big sur causing these problems?

**Comment by Saviq** [comment]
No, it shouldn't be. And `multipass purge` is really just about removing some files and data, so it's not clear what could go wrong there.
